deploy/manifests/virtual-kubelet/base/statefulset.yaml

```diff
@@ -149,3 +162,12 @@ spec:
         hostPath:
           path: /lib/modules
           type: ""
+  volumeClaimTemplates:
+    - metadata:
+        name: data
+      spec:
+        accessModes: 
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
```

Comment:
Can we increase the size of this?  Typically our etcd data is ~ 500MB, even for small deployments.  We might want to go anywhere from 3-5GB, an etcd volume can grow up to 10GB.